wordlists/names/top_male_names_turkish.txt

```diff
@@ -0,0 +1,873 @@
+Adem
+Adem
+Ahmet
+Ahmet
+Ahmet
+Akın
+Ali
+Ali
+Ali
+Alp
+Alparslan
+Alper
+Alper
```

Comment:
Duplicates

Reply:
ok I am removing all the duplicates

Reply:
Sorry for the inconvenience. I have updated and there are no duplicates.